requirements.txt

```diff
@@ -1,10 +1,11 @@
 colorama
-gensim
 pandas
 scikit-learn==1.3.1
 nltk
-torch
-transformers
+gensim>=4.1.0
```

Comment:
What do we need gensim for in the project?

Reply:
Not sure